build/commands/scripts/commands.js

```diff
@@ -67,6 +67,32 @@ program
   .arguments('[build_config]')
   .action(applyPatches)
 
+program
+  .command('update_symlink')
+  .option('--symlink_dir <symlink_dir>', 'symlink that points to the actual build directory')
+  .option('--target_os <target_os_type>', 'target OS type', /^(host_os|ios|android)$/i)
+  .option('--target_arch <target_arch>', 'target architecture', /^(host_cpu|x64|arm64|x86)$/i)
+  .arguments('[build_config]')
+  .action((buildConfig = config.defaultBuildConfig, options) => {
+    config.buildConfig = buildConfig
+    if (options.target_os == 'host_os')
+      delete options.target_os
+
+    if (options.target_arch == 'host_cpu')
+      delete options.target_arch
+
+    config.update(options)
+    const current_link = options.symlink_dir
+    if (!path.isAbsolute(current_link) && !path.relative(current_link, config.srcDir).startsWith('..')) {
+      console.error('Symlink must be an absolute path in src')
+      process.exit(1)
+    }
+
+    fs.removeSync(current_link)
+    fs.symlinkSync(config.outputDir, current_link)
```

Comment:
make it `junction` on windows. https://github.com/nodejs/node/issues/22208#issuecomment-605550545